### ISSUE TITLE
Typescript: import autocomplete fills in function arguments

### ISSUE THREAD

**Issue by andy-hanson** [Microsoft/vscode#2092]
When I use autocomplete to import a function, it fills in the function arguments right there.

Here's what it looks like:
- `import {foo} from './fooBar'`
- `import {foo, b} from './fooBar'`
- `bar` shows up as a suggesstion; hit enter.
- It fills in as `import {foo, bar(argname)} from './fooBar'`.

**Comment by dbaeumer** [comment]
@andy-hanson I assume you have set 	"typescript.useCodeSnippetsOnMethodSuggest": true. Right?

**Comment by andy-hanson** [comment]
Right.

**Comment by mjbvz** [comment]
Quick note: this is not limited to imports, any assignment can be impacted:

```js
const a = {
    func(x) {}
}

let {f$} = a;
```

completes to 

```js
let {func(x)} = a;
````

**Comment by zakdances** [comment]
I'm still getting this error. The autocomplete just fills in an argument's type, not a correct signature.

**Comment by mjbvz** [comment]
@zakdances Are you on 1.9 insiders? This fix is not in VSCode 1.8.

**Comment by zakdances** [comment]
@mjbvz I'm using 1.8.1 so I guess you can disregard my message. Thanks for the heads up!
